packages/tocco-util/src/notifier/notificationActionFactory.js

```diff
@@ -61,16 +60,12 @@ export function getBlockingInfo(id, title, message, icon) {
   const options = {
     timeOut: 0,
     showCloseButton: false,
-    transitionIn: 'bounceInDown',
-    transitionOut: 'bounceOutUp',
     attention: true,
     onAttentionClick: () => {},
     component: () => <TitleMessage title={title} message={message}/>
   }
 
-  if (icon) {
-    options.icon = <Icon icon={icon} size="2x" />
-  }
+  options.icon = _isString(icon) ? <Icon icon={`${icon}`} size="3x" /> : icon
```

Comment:
icon={icon} ?

Reply:
What if icon is undefined? Does that work?

Reply:
If icon is undefined Toastr's default icons are used. Instead of leaving options.icon undefined I set a default icon.